### DISCUSSION TITLE
How to add "vendor_available" option of Android.bp in CMake?

### DISCUSSION THREAD

**Discussion by yoloisvtyois** [android/ndk#1701] [Q&A] [answered]
I'm building a library using CMake and need to set "vendor_available" property of Android.bp (or "LOCAL_VENDOR_MODULE" of Android.mk) with using CMake. Is there a way to do it?

**Comment by enh-google** [comment, marked as answer]
no. we only support cmake for building *apps*. you'll need to use the platform build system for OS code.
